Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -2525,49 +2647,6 @@ void RenderOpenGL::RemoveTextureFromDevice(Texture* texture) { __debugbreak(); }
 
 bool RenderOpenGL::MoveTextureToDevice(Texture *texture) {
     auto t = (TextureOpenGL *)texture;
-    /*    return false;
```

Comment:
sorry just a side note, not related to this PR
if you check original game and try to switch d3d vs soft render - you could see that different monster trier gets different coloring (usually green, blue, red depending on monster level) - but in soft it's done via palettes and is pretty nice while in d3d the entire texture just gets tinted. i think with shaders we can emulate that nice softrender touch
please let me know if the explanation is not good enough - i can show you examples

Reply:
Yes i know what you mean. The higher resolution D3D sprites have fixed colours. Could definitely emulate the palette effect, but would have to use the lower resolution sprites to do so i think.